auto_dev/protocols/performatives.py

```diff
@@ -1,8 +1,9 @@
+"""Module for parsing protocol performatives."""
 
 
 SCALAR_MAP = {
-    "int": "Int64",
-    "float": "Double",
+    "int": "conint(ge=Int32.min(), le=Int32.max())",
+    "float": "confloat(ge=Double.min(), le=Double.max())",
```

Comment:
The framework does not accept a subclass of `int` or `float` in it's consistency check for message content. However, it also does not accept that it checks for should be there! Both `int` and `float` need to be in the protobuf defined range otherwise this won't pass tests for the performatives. 

You may also appreciate that for the python builtin `int` 32 bits have been allocated, whereas for builtin `float` 64 bits have been allocated: see [PYTHON_TYPE_TO_PROTO_TYPE](https://github.com/valory-xyz/open-aea/blob/26738e20ca0937b5695e43bf2267525a475d8af4/aea/protocols/generator/common.py#L69-L75)